frontend/src/features/map_planning/layers/drawing/actions.ts

```diff
@@ -0,0 +1,304 @@
+/**
```

Comment:
Consider adding unit tests for all actions frontend.
Contrary to most of our UI code, actions are relatively easy to test with a high return on investment due to the complex nature of our frontend state management. 

You can take a look at my ShadeLayer branch for an example of how this can be done.

Reply:
I added a new issue to add tests: https://github.com/ElektraInitiative/PermaplanT/issues/1222